workspace.json

```diff
@@ -1866,7 +1866,7 @@
               "apps/application-system/form/src/favicon.ico",
               "apps/application-system/form/src/assets"
             ],
-            "styles": [],
+            "styles": ["apps/application-system/form/src/styles.scss"],
```

Comment:
Could the logic be done here: `libs/island-ui/core/src/global/fonts.treat.ts`

and above
```
"assets" : [
  ...,
  libs/island-ui/core/src/assets/fonts
]
```

this is the theory, not sure if it works

Reply:
I believe there is an open issue on the Treat repo that addresses this and according to them, setting global font-face in Treat does not work as expected. I'll post this on Slack to see if anyone has the answer to this.

https://github.com/seek-oss/treat/issues/93

Reply:
Then I am thinking this should be resolved in a later PR :)